Sources/Harmony/Data/DataSource/PutNetworkDataSource.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  PutNetworkDataSource.swift
+//  Harmony
+//
+//  Created by Borja Arias Drake on 11.10.2022..
+//
+
+import Foundation
+import Alamofire
```

Comment:
why do we have a dependency on Alamofire? URLSession is powerful enough to be our base networking class. I wouldn't add any dependencies to a library as Alamofire as we need to maintain it. What is the reason behind this coupling? and why it can't be avoided? (can it?)

Reply:
I mean, why complicating our lives with a dependency to a huge library like this, while we can use directly the built-in URLSession and avoid an extra dependency.

Reply:
I am not aware of why it got introduced. I prefer not to use frameworks that just wrap Apple's ones. 

I will remove it.

Reply:
https://github.com/mobilejazz/harmony-swift/pull/48

Reply:
I checked and approved the #48 PR. Maybe @vilanovi you can also could take a look.